CHANGELOG.rst

```diff
@@ -4,6 +4,31 @@ CHANGELOG
 This document describes changes between each past release as well as
 the version control of each dependency.
 
+26.0.0 (unreleased) 
+===================
+
+**Breaking Changes**
+
+- Do not install ``kinto_changes`` and ``kinto_signer`` plugins from PyPI.
+  The source code for these plugins has moved to this repository and will
+  continue to be maintained here. ``kinto-changes`` was copied to this repo at
+  commit 0af75a9efb6ae849ee1a6761349bfaf49dd1c488, and ``kinto-signer`` was
+  copied at commit 249db348caec02daafc4c249658b4ca2a89343bd. After they were
+  copied, linting and formatting tools were run against them and changes were
+  made for CI checks to pass. 
+
+- ``kinto_signer``'s internal package ``signer`` was renamed to ``backends``
+
+- ``kinto_changes`` and ``kinto_signer`` are now combined into one plugin
+  ``kinto_remote_settings``. Kinto config files must be changed to register the
+  plugin as such. 
+  
+- Some config files (e.g. Remote Settings server configuration, `permissions
+  metadata <https://github.com/mozilla-services/remote-settings-permissions>`_.)
+  rely on classes from ``kinto_signer`` and ``kinto_changes`` to have specific
+  qualified names. These values must be changed to correctly refer to these
+  classes.
```

Comment:
Let's be exhaustive here. 

The config part could be covered with the rename of signer_backends above. Hence we could focus on the metadata.
It is actually the «collections metadata», that is read by kinto-emailer. And more precisely the event selector in the collection metadata. So we could write this:

```rst
Some collection metadata rely on classes from ``kinto_signer`` for kinto-emailer to send emails on signer events (review requests, approvals, ...) and must thus be changed to correctly refer to these classes. For example:

Before:

.. code-block:: yaml

   kinto-emailer:
        hooks:
        - event: kinto_signer.events.ReviewRequested

After:

.. code-block:: yaml

   kinto-emailer:
        hooks:
        - event: kinto_remote_settings.signer.events.ReviewRequested

```